datumaro/util/telemetry_utils.py

```diff
@@ -0,0 +1,142 @@
+    elif command is contexts.project.transform_command:
+        return 'project_transform_result'
+    elif command is contexts.project.info_command:
+        return 'project_info_result'
+    elif command is contexts.project.stats_command:
+        return 'project_stats_result'
+    elif command is contexts.project.validate_command:
+        return 'project_validate_result'
+    elif command is contexts.project.migrate_command:
+        return 'project_migrate_result'
+    elif command is contexts.source.add_command:
+        return 'source_add_result'
+    elif command is contexts.source.remove_command:
+        return 'source_remove_result'
+    elif command is contexts.source.info_command:
+        return 'source_info_result'
+    elif command is contexts.model.add_command:
+        return 'model_add_result'
+    elif command is contexts.model.remove_command:
+        return 'model_remove_result'
+    elif command is contexts.model.run_command:
+        return 'model_run_result'
+    elif command is contexts.model.info_command:
+        return 'model_info_result'
+    elif command is commands.checkout.checkout_command:
+        return 'checkout_result'
+    elif command is commands.commit.commit_command:
+        return 'commit_result'
+    elif command is commands.convert.convert_command:
+        return 'convert_result'
+    elif command is commands.create.create_command:
+        return 'create_result'
+    elif command is commands.diff.diff_command:
+        return 'diff_result'
+    elif command is commands.explain.explain_command:
+        return 'explain_result'
+    elif command is commands.info.info_command:
+        return 'info_result'
+    elif command is commands.log.log_command:
+        return 'log_result'
+    elif command is commands.merge.merge_command:
+        return 'merge_result'
+    elif command is commands.status.status_command:
+        return 'status_result'
+
+    return 'unknown_command_result'
+
+def _cleanup_params_info(args, sensitive_args):
+    fields_to_exclude = ('command', '_positionals',)
+    cli_params = {}
+    for arg in vars(args):
+        if arg in fields_to_exclude:
+            continue
+        arg_value = getattr(args, arg)
+        if arg in sensitive_args:
+            # If command line argument value is a directory or a path to file it is not sent
+            # as it may contain confidential information. "1" value is used instead.
+            cli_params[arg] = str(1)
+        else:
+            cli_params[arg] = str(arg_value)
+    return cli_params
+
+def _cleanup_stacktrace():
+    installation_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    def clean_path(match):
+        file_path = match.group(1)
+        if file_path.startswith(installation_dir):
+            file_path = os.path.relpath(file_path, installation_dir)
+        else:
+            file_path = os.path.basename(file_path)
+
+        return 'File "{}"'.format(file_path)
```

Comment:
```suggestion
        return f"File \"{file_path}\""
```
We don't use the `.format` syntax.